packages/foreman/foreman/foreman.spec

```diff
@@ -691,6 +693,9 @@ plugins required for Foreman to work.
   SYSCONFDIR=%{_sysconfdir} \
   --trace
 
+#use SELinux wrapper for sidekiq
+sed -i '/^ExecStart/ s|/usr/bin/sidekiq \(.\+\)$|/usr/share/foreman/bin/foreman-sidekiq \1|' extras/systemd/%{dynflow_sidekiq_service_name}.service
```

Comment:
I'd use the same macros as you use in the `install` line so you're sure they're in sync.

Reply:
Sorry what macros? Not following.